lib/mix/tasks/thesis.install.ex

```diff
@@ -137,16 +138,13 @@ defmodule Mix.Tasks.Thesis.Install do
         store: Thesis.EctoStore,
         authorization: #{Mix.Phoenix.base}.ThesisAuth,
         uploader: Thesis.RepoUploader
-        #  uploader: <MyApp>.<CustomUploaderModule>
-        #  uploader: Thesis.OspryUploader
       config :thesis, Thesis.EctoStore, repo: #{Mix.Phoenix.base}.Repo
-      # config :thesis, Thesis.OspryUploader,
-      #   ospry_public_key: "pk-prod-asdfasdfasdfasdf"
-      # If you want to allow creating dynamic pages:
+
+      # If you want to allow creating dynamic pages, enable and customize this:
       # config :thesis, :dynamic_pages,
-      #   view: #{Mix.Phoenix.base}.PageView,
+      #   view: #{Mix.Phoenix.base}Web.PageView,
       #   templates: ["index.html", "otherview.html"],
-      #   not_found_view: #{Mix.Phoenix.base}.ErrorView,
+      #   not_found_view: #{Mix.Phoenix.base}Web.ErrorView,
```

Comment:
The install script won't work out of the box with Phoenix 1.2 anymore. I'm not too sad about it.

Reply:
How does this affect existing sites running on phoenix 1.2 that need new migrations via the install script?

Reply:
It shouldn't affect those.